frontend/public/lib/selectOptionsOperations.ts

```diff
@@ -26,6 +26,7 @@ export function parseOptions(options, parentPropertyName) {
 }
 
 export function parseSectorOptions(options) {
+  // TODO: should we move this to lib/selectOptionsOperations.ts?
```

Comment:
It's already there?

Reply:
xD sorry, I meant to say
> should we move this to `lib/sectorOperations.ts`
It is a bit wired, because, for the backend, there is no `SectorOption` type. There are only `Sector`s.